enterprise/frontend/src/metabase-enterprise/content_verification/ModelFilterControls.tsx

```diff
@@ -60,3 +66,17 @@ export const ModelFilterControls = ({
     </Popover>
   );
 };
+
+const Dot = () => {
```

Comment:
How would you feel about just using an `<Icon />` rather than a custom component for this? We do have a `"recents"` icon that is just a filled circle (though that's a super confusing name).

Reply:
I'd slightly prefer to use this since it's dead simple and not coupled to something conceptually unrelated, that could hypothetically change later

Reply:
I think the correct thing to do here is to add a `dot` icon to the icon library and use that. But if you'd rather not then I won't stop you from merging the PR